src/Core/Framework/Webhook/WebhookDispatcher.php

```diff
@@ -73,13 +73,10 @@ public function dispatch(object $event, ?string $eventName = null): object
             return $event;
         }
 
-        foreach ($this->eventFactory->createHookablesFor($event) as $hookable) {
-            $context = Context::createDefaultContext();
-            if ($event instanceof FlowEventAware || $event instanceof AppChangedEvent || $event instanceof EntityWrittenContainerEvent) {
-                $context = $event->getContext();
-            }
+        $context = Context::createDefaultContext();
 
-            $this->callWebhooks($hookable, $context);
+        foreach ($this->eventFactory->createHookablesFor($event) as $hookable) {
+            $this->callWebhooks($hookable, $event instanceof FlowEventAware || $event instanceof AppChangedEvent || $event instanceof EntityWrittenContainerEvent ? $event->getContext() : $context);
```

Comment:
this was better readable before :grin:

Reply:
I changed it again, it should be easier to read now.